2025-03-09 22:10:13,164 INFO    Thread-11 (_run_job):4620 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221013-5pg48f66\logs\debug.log
2025-03-09 22:10:13,166 INFO    Thread-11 (_run_job):4620 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221013-5pg48f66\logs\debug-internal.log
2025-03-09 22:10:13,166 INFO    Thread-11 (_run_job):4620 [wandb_init.py:init():761] calling init triggers
2025-03-09 22:10:13,167 INFO    Thread-11 (_run_job):4620 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'sigmoid', 'batchSize': 132, 'hiddenLayer_Size': 128, 'learningRate': 0.0001, 'num_epochs': 5, 'num_hiddenLayers': 5, 'optimizer': 'nadam', 'weightDecay': 0, 'weightInit': 'random'}
config: {'num_epochs': 10, 'num_hiddenLayers': 3, 'hiddenLayer_Size': 40, 'weightDecay': 0, 'learningRate': 0.001, 'optimizer': 'Nesterov_Accelerated_GD', 'batchSize': 32, 'weightInit': 'xavier', 'activationFunc': 'sigmoid', '_wandb': {}}
2025-03-09 22:10:13,167 INFO    Thread-11 (_run_job):4620 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-09 22:10:13,308 INFO    Thread-11 (_run_job):4620 [wandb_run.py:_finish():2112] finishing run da6401-assignments/assignment1/5pg48f66
2025-03-09 22:10:13,313 ERROR   Thread-11 (_run_job):4620 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-09 22:10:13,313 INFO    Thread-11 (_run_job):4620 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 22:10:13,313 INFO    Thread-11 (_run_job):4620 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-09 22:10:13,313 INFO    Thread-11 (_run_job):4620 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 1
2025-03-09 22:10:13,313 INFO    Thread-11 (_run_job):4620 [wandb_run.py:_restore():2322] restore
2025-03-09 22:10:13,314 INFO    Thread-11 (_run_job):4620 [wandb_run.py:_restore():2328] restore done
2025-03-09 22:10:15,373 INFO    Thread-11 (_run_job):4620 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-09 22:10:15,373 INFO    Thread-11 (_run_job):4620 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-09 22:10:15,373 INFO    Thread-11 (_run_job):4620 [wandb_run.py:_footer_sync_info():3917] logging synced files
2025-03-09 22:10:15,377 ERROR   MainThread:4620 [pyagent.py:_run_jobs_from_queue():234] Run 5pg48f66 errored:
Traceback (most recent call last):
  File "c:\Python312\Lib\site-packages\wandb\agents\pyagent.py", line 306, in _run_job
    self._function()
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\1058529509.py", line 36, in sweep_hyperParameters
    model.train(x_train, y_train, x_val, y_val, num_epochs, learningRate, optimizer, batchSize)
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\3882185316.py", line 60, in train
    self.forward(x_batch)
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\3882185316.py", line 107, in forward
    x = self.sigmoid(np.matmul(x.T, w) + b)
                     ^^^^^^^^^^^^^^^^^
ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 128 is different from 132)

2025-03-09 22:10:20,398 INFO    MsgRouterThr:4620 [mailbox.py:close():129] Closing mailbox, abandoning 1 handles.
2025-03-09 22:10:21,529 INFO    Thread-16 (_run_job):4620 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-09 22:10:21,529 INFO    Thread-16 (_run_job):4620 [wandb_setup.py:_flush():67] Configure stats pid to 4620
2025-03-09 22:10:21,529 INFO    Thread-16 (_run_job):4620 [wandb_setup.py:_flush():67] Loading settings from C:\Users\prajv\.config\wandb\settings
2025-03-09 22:10:21,529 INFO    Thread-16 (_run_job):4620 [wandb_setup.py:_flush():67] Loading settings from c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\settings
2025-03-09 22:10:21,529 INFO    Thread-16 (_run_job):4620 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-09 22:10:21,529 INFO    Thread-16 (_run_job):4620 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221021-mvmw2t0x\logs\debug.log
2025-03-09 22:10:21,549 INFO    Thread-16 (_run_job):4620 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221021-mvmw2t0x\logs\debug-internal.log
2025-03-09 22:10:21,549 INFO    Thread-16 (_run_job):4620 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000024D42ACBCB0>
2025-03-09 22:10:21,549 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():761] calling init triggers
2025-03-09 22:10:21,549 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'relu', 'batchSize': 132, 'hiddenLayer_Size': 128, 'learningRate': 0.001, 'num_epochs': 5, 'num_hiddenLayers': 5, 'optimizer': 'rmsprop', 'weightDecay': 0, 'weightInit': 'xavier'}
config: {'_wandb': {}}
2025-03-09 22:10:21,549 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():784] starting backend
2025-03-09 22:10:21,549 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():788] sending inform_init request
2025-03-09 22:10:21,582 INFO    Thread-16 (_run_job):4620 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-03-09 22:10:21,612 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():798] backend started and connected
2025-03-09 22:10:21,614 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_config_callback():1261] config_cb None None {'activationFunc': 'relu', 'batchSize': 132, 'hiddenLayer_Size': 128, 'learningRate': 0.001, 'num_epochs': 5, 'num_hiddenLayers': 5, 'optimizer': 'rmsprop', 'weightDecay': 0, 'weightInit': 'xavier'}
2025-03-09 22:10:21,617 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-09 22:10:21,637 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():891] updated telemetry
2025-03-09 22:10:21,701 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-09 22:10:23,370 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():990] starting run threads in backend
2025-03-09 22:10:23,598 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_console_start():2375] atexit reg
2025-03-09 22:10:23,598 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-09 22:10:23,598 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-09 22:10:23,598 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-09 22:10:23,599 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-09 22:10:23,604 INFO    Thread-16 (_run_job):4620 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221023-mvmw2t0x\logs\debug.log
2025-03-09 22:10:23,604 INFO    Thread-16 (_run_job):4620 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221023-mvmw2t0x\logs\debug-internal.log
2025-03-09 22:10:23,604 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():761] calling init triggers
2025-03-09 22:10:23,604 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'relu', 'batchSize': 132, 'hiddenLayer_Size': 128, 'learningRate': 0.001, 'num_epochs': 5, 'num_hiddenLayers': 5, 'optimizer': 'rmsprop', 'weightDecay': 0, 'weightInit': 'xavier'}
config: {'num_epochs': 10, 'num_hiddenLayers': 3, 'hiddenLayer_Size': 40, 'weightDecay': 0, 'learningRate': 0.001, 'optimizer': 'Nesterov_Accelerated_GD', 'batchSize': 32, 'weightInit': 'xavier', 'activationFunc': 'sigmoid', '_wandb': {}}
2025-03-09 22:10:23,604 INFO    Thread-16 (_run_job):4620 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-09 22:10:23,745 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_finish():2112] finishing run da6401-assignments/assignment1/mvmw2t0x
2025-03-09 22:10:23,746 ERROR   Thread-16 (_run_job):4620 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-09 22:10:23,746 INFO    Thread-16 (_run_job):4620 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 22:10:23,746 INFO    Thread-16 (_run_job):4620 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-09 22:10:23,746 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 1
2025-03-09 22:10:23,746 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_restore():2322] restore
2025-03-09 22:10:23,746 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_restore():2328] restore done
2025-03-09 22:10:25,728 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-09 22:10:25,728 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-09 22:10:25,728 INFO    Thread-16 (_run_job):4620 [wandb_run.py:_footer_sync_info():3917] logging synced files
2025-03-09 22:10:25,743 ERROR   MainThread:4620 [pyagent.py:_run_jobs_from_queue():234] Run mvmw2t0x errored:
Traceback (most recent call last):
  File "c:\Python312\Lib\site-packages\wandb\agents\pyagent.py", line 306, in _run_job
    self._function()
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\1058529509.py", line 36, in sweep_hyperParameters
    model.train(x_train, y_train, x_val, y_val, num_epochs, learningRate, optimizer, batchSize)
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\3882185316.py", line 60, in train
    self.forward(x_batch)
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\3882185316.py", line 107, in forward
    x = self.sigmoid(np.matmul(x.T, w) + b)
                     ^^^^^^^^^^^^^^^^^
ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 128 is different from 132)

2025-03-09 22:10:35,810 INFO    MsgRouterThr:4620 [mailbox.py:close():129] Closing mailbox, abandoning 1 handles.
2025-03-09 22:10:36,877 INFO    Thread-21 (_run_job):4620 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-09 22:10:36,877 INFO    Thread-21 (_run_job):4620 [wandb_setup.py:_flush():67] Configure stats pid to 4620
2025-03-09 22:10:36,877 INFO    Thread-21 (_run_job):4620 [wandb_setup.py:_flush():67] Loading settings from C:\Users\prajv\.config\wandb\settings
2025-03-09 22:10:36,877 INFO    Thread-21 (_run_job):4620 [wandb_setup.py:_flush():67] Loading settings from c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\settings
2025-03-09 22:10:36,877 INFO    Thread-21 (_run_job):4620 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-09 22:10:36,885 INFO    Thread-21 (_run_job):4620 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221036-uzj0zct4\logs\debug.log
2025-03-09 22:10:36,886 INFO    Thread-21 (_run_job):4620 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221036-uzj0zct4\logs\debug-internal.log
2025-03-09 22:10:36,886 INFO    Thread-21 (_run_job):4620 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000024D42B38320>
2025-03-09 22:10:36,886 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():761] calling init triggers
2025-03-09 22:10:36,886 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'sigmoid', 'batchSize': 132, 'hiddenLayer_Size': 128, 'learningRate': 0.001, 'num_epochs': 20, 'num_hiddenLayers': 3, 'optimizer': 'rmsprop', 'weightDecay': 0.5, 'weightInit': 'xavier'}
config: {'_wandb': {}}
2025-03-09 22:10:36,886 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():784] starting backend
2025-03-09 22:10:36,886 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():788] sending inform_init request
2025-03-09 22:10:36,886 INFO    Thread-21 (_run_job):4620 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-03-09 22:10:36,891 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():798] backend started and connected
2025-03-09 22:10:36,892 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_config_callback():1261] config_cb None None {'activationFunc': 'sigmoid', 'batchSize': 132, 'hiddenLayer_Size': 128, 'learningRate': 0.001, 'num_epochs': 20, 'num_hiddenLayers': 3, 'optimizer': 'rmsprop', 'weightDecay': 0.5, 'weightInit': 'xavier'}
2025-03-09 22:10:36,894 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-09 22:10:36,901 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():891] updated telemetry
2025-03-09 22:10:36,947 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-09 22:10:38,522 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():990] starting run threads in backend
2025-03-09 22:10:38,784 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_console_start():2375] atexit reg
2025-03-09 22:10:38,785 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-09 22:10:38,786 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-09 22:10:38,786 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-09 22:10:38,788 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-09 22:10:38,795 INFO    Thread-21 (_run_job):4620 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221038-uzj0zct4\logs\debug.log
2025-03-09 22:10:38,795 INFO    Thread-21 (_run_job):4620 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_221038-uzj0zct4\logs\debug-internal.log
2025-03-09 22:10:38,797 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():761] calling init triggers
2025-03-09 22:10:38,797 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'sigmoid', 'batchSize': 132, 'hiddenLayer_Size': 128, 'learningRate': 0.001, 'num_epochs': 20, 'num_hiddenLayers': 3, 'optimizer': 'rmsprop', 'weightDecay': 0.5, 'weightInit': 'xavier'}
config: {'num_epochs': 10, 'num_hiddenLayers': 3, 'hiddenLayer_Size': 40, 'weightDecay': 0, 'learningRate': 0.001, 'optimizer': 'Nesterov_Accelerated_GD', 'batchSize': 32, 'weightInit': 'xavier', 'activationFunc': 'sigmoid', '_wandb': {}}
2025-03-09 22:10:38,797 INFO    Thread-21 (_run_job):4620 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-09 22:10:38,978 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_finish():2112] finishing run da6401-assignments/assignment1/uzj0zct4
2025-03-09 22:10:38,982 ERROR   Thread-21 (_run_job):4620 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-09 22:10:38,982 INFO    Thread-21 (_run_job):4620 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 22:10:38,985 INFO    Thread-21 (_run_job):4620 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-09 22:10:38,986 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 1
2025-03-09 22:10:38,986 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_restore():2322] restore
2025-03-09 22:10:38,986 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_restore():2328] restore done
2025-03-09 22:10:40,979 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-09 22:10:40,979 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-09 22:10:40,979 INFO    Thread-21 (_run_job):4620 [wandb_run.py:_footer_sync_info():3917] logging synced files
2025-03-09 22:10:40,984 ERROR   MainThread:4620 [pyagent.py:_run_jobs_from_queue():234] Run uzj0zct4 errored:
Traceback (most recent call last):
  File "c:\Python312\Lib\site-packages\wandb\agents\pyagent.py", line 306, in _run_job
    self._function()
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\1058529509.py", line 36, in sweep_hyperParameters
    model.train(x_train, y_train, x_val, y_val, num_epochs, learningRate, optimizer, batchSize)
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\3882185316.py", line 60, in train
    self.forward(x_batch)
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_4620\3882185316.py", line 107, in forward
    x = self.sigmoid(np.matmul(x.T, w) + b)
                     ^^^^^^^^^^^^^^^^^
ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 128 is different from 132)

2025-03-09 22:10:40,985 ERROR   MainThread:4620 [pyagent.py:_run_jobs_from_queue():245] Detected 3 failed runs in the first 60 seconds, killing sweep.
